R/tar-terra-sprc.R

```diff
@@ -62,6 +62,10 @@ tar_terra_sprc <- function(name,
                            retrieval = targets::tar_option_get("retrieval"),
                            cue = targets::tar_option_get("cue")) {
   check_pkg_installed("terra")
+    #ensure that user-passed `resources` doesn't include `custom_format`
+    if ("custom_format" %in% names(resources)) {
+        stop("`custom_format` cannot be supplied to targets created with `tar_terra_sprc()`")
+    }
```

Comment:
I like the suggestion to use `cli`, but the additional function seems like overkill since I don't anticipate needing to check for other things in resources besides custom_format